tx-pool/src/pool.rs

```diff
@@ -621,27 +486,27 @@ impl TxPool {
         }
         (entries, size, cycles)
     }
-}
 
-fn build_recent_reject(config: &TxPoolConfig) -> Option<RecentReject> {
-    if !config.recent_reject.as_os_str().is_empty() {
-        let recent_reject_ttl = config.keep_rejected_tx_hashes_days as i32 * 24 * 60 * 60;
-        match RecentReject::new(
-            &config.recent_reject,
-            config.keep_rejected_tx_hashes_count,
-            recent_reject_ttl,
-        ) {
-            Ok(recent_reject) => Some(recent_reject),
-            Err(err) => {
-                error!(
-                    "Failed to open recent reject database {:?} {}",
-                    config.recent_reject, err
-                );
-                None
+    fn build_recent_reject(config: &TxPoolConfig) -> Option<RecentReject> {
+        if !config.recent_reject.as_os_str().is_empty() {
+            let recent_reject_ttl = config.keep_rejected_tx_hashes_days as i32 * 24 * 60 * 60;
```

Comment:
Do we need to check `config.keep_rejected_tx_hashes_days` is not `0`?

Reply:
It is better to have a minimum value `max(1, config.keep_rejected_tx_hashes_days)`.